content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md

```diff
@@ -0,0 +1,172 @@
+---
+title: RUM Unity Monitoring Setup
+kind: documentation
+is_beta: true
+private: true
+disable_toc: false
+description: Collect RUM data from your Unity Mobile projects.
+aliases:
+    - /real_user_monitoring/unity/
+    - /real_user_monitoring/unity/setup
+code_lang: unity
+type: multi-code-lang
+code_lang_weight: 30
+further_reading:
+- link: https://github.com/DataDog/dd-sdk-unity
+  tag: GitHub
+  text: Source code for dd-sdk-unity
+- link: https://github.com/DataDog/unity-package
+  tag: GitHub
+  text: Package URL for Unity SDK
+- link: real_user_monitoring/explorer/
+  tag: Documentation
+  text: Learn how to explore your RUM data
+- link: https://www.datadoghq.com/blog/monitor-flutter-application-performance-with-mobile-rum/
+  tag: Blog
+  text: Monitor Flutter application performance with Datadog Mobile RUM
+
+---
+## Overview
+
+{{< beta-callout url="#" btn_hidden="true" >}}
+Unity Monitoring is currently in private beta.
+{{< /beta-callout >}}
+
+Datadog Real User Monitoring (RUM) enables you to visualize and analyze user journeys of your application's individual users.
+
+## Setup
+
+<div class="alert alert-info">
+Datadog supports Unity Monitoring for iOS and Android for Unity LTS 2022+
+</div>
+
+Datadog does not currently support Desktop (Windows, Mac, or Linux), console or Web deployments from Unity, but Datadog's priorities are often re-evaluated based on your feedback. If you have a game or application and would want to use Datadog RUM to monitor its performance, reach out to your customer support team and escalate this feature request.
```

Comment:
```suggestion
Datadog does not support Desktop (Windows, Mac, or Linux), console, or web deployments from Unity. If you have a game or application and want to use Datadog RUM to monitor its performance, create a ticket with [Datadog support](/help/).
```

Reply:
Is there any specific information the customer could provide that would be helpful to start? Here's an example: https://docs.datadoghq.com/integrations/guide/requests/#submit-a-request

Reply:
The only think I can think of would be what platform they want us to support.